scripts/verify_secure_store.py

```
@@ -0,0 +1,192 @@
+k3 = derive_key("machine-a", "/home/v")
+check("key 长度 32", len(k1) == 32)
+check("不同 machine-id 派生不同 key", k1 != k2)
+check("不同 HOME 派生不同 key", k1 != k3)
+
+print("== 3. 加密→解密 往返（模拟 C++ 完整链路）==")
+MID, HOME = "9d27f8c1e2a34b5c8f0e1234567890ab", "/home/user"
+for plain in ["sk-1234567890abcdef", "sk-abc", "", "a" * 100, "中文密钥测试 🔑"]:
+    enc = encrypt_cpp(plain, MID, HOME)
+    dec = decrypt_cpp(enc, MID, HOME)
+    if plain == "":
+        check(f"空串不加密: enc==''", enc == "")
```

Flake8 (7.3.0)

[error] f-string is missing placeholders

(F541)